bin/servicerouter.py

```diff
@@ -27,85 +29,87 @@
 Usage:
   $ servicerouter.py --marathon http://marathon1:8080 --haproxy-config /etc/haproxy/haproxy.cfg
 
-  The user servicerouter is executed as must have the right to reload haproxy.
+  The user that executes servicerouter must have the permission to reload
+  haproxy.
 
 
 Environment Variables:
   HAPROXY_GROUP
-  Only servicerouter instances which are members of the given group will
-  point to the service. Service routers with the group '*' will collect
-      all groups.
+    The group of servicerouter instances that point to the service.
+    Service routers with the group '*' will collect all groups.
 
   HAPROXY_{n}_VHOST
-  HTTP Virtual Host proxy hostname to catch
-  Ex: HAPROXY_0_VHOST = 'marathon.mesosphere.com'
+    The Marathon HTTP Virtual Host proxy hostname to gather.
```

Comment:
I like neither catch nor gather a lot here.

What is meant is that HAProxy will accept requests on this virtual host and forward them to tasks which have this HAPROXY_{n}_VHOST setting, correct?

I would also mention that you can specify multiple host names and extend the example accordingly.